ext/mixed/js/display.js

```diff
@@ -269,7 +269,9 @@ class Display {
 
     async termsShow(definitions, options, context) {
         try {
-            window.focus();
+            if (context && context.focus) {
```

Comment:
This could cause a bug when context is null or undefined. Maybe replace with `if (!context || context.focus) {`, or `if (!context || context.type === 'mouse') {` if you do the search type check later

Reply:
Fixed.